src/reconcile/rec_hs.c

```diff
@@ -694,7 +695,14 @@ __wti_rec_hs_insert_updates(WT_SESSION_IMPL *session, WTI_RECONCILE *r, WT_MULTI
         if (list->onpage_upd == NULL)
             continue;
 
-        /* Skip aborted updates. */
+        if (__wt_txn_upd_visible_all(session, list->onpage_upd))
```

Comment:
Remove this check leads to a lot of test format failure which I don't understand. We will see a modify update without a full value.

Reply:
Then is this an existing problem?